dev/dll/Microsoft.UI.Xaml.vcxproj

```diff
@@ -386,6 +383,9 @@
     <Natvis Include="$(IntermediateOutputPath)CppWinRT\Component\winrt\Microsoft.UI.Xaml.natvis">
     </Natvis>
     -->
+    <ClCompile Update="C:\Users\t-ketaha\source\repos\microsoft-ui-xaml\dev\NumberBox\..\..\..\..\..\Code\tinyexpr\NumberBoxExpressionEvaluator.c">
```

Comment:
Should this file be part of this repo/commit?

Reply:
Ah this was from when I had a substitute external library. Will remove, thanks!

Reply:
Ah this was from when I had a substitute external library. Will remove, thanks!